pep-9999.rst

```diff
@@ -87,6 +87,16 @@ Supported Constructs
 This sections lists constructs that compliant type checkers are expected
 to accept. Type stub authors can safely use these constructs.
 
+Structured comments
+-------------------
+
+Two kinds of structured comments are accepted:
+
+* A ``# type: X`` comment at the end of a line that defines a variable,
+  declaring that the variable has type ``X``.
```

Comment:
Yes, in fact we got rid of (almost?) all non-ignore type comments from typeshed, and we shouldn't accept new ones. I'd prefer that the PEP says that you should always use variable annotations.

Reply:
Done. The style guide section already says to use the newest language features and gives an example of using a variable annotation rather than a type comment.